boost-maven/boost-runtimes/runtime-openliberty/pom.xml

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-	<groupId>boost.runtimes</groupId>
-	<artifactId>openliberty</artifactId>
-	<version>0.1-SNAPSHOT</version>
+    <groupId>boost.runtimes</groupId>
+    <artifactId>openliberty</artifactId>
+    <version>0.1-SNAPSHOT</version>
```

Comment:
The **0.1-SNAPSHOT** servers the purpose of separtely versioning from the pluging..but maybe 1.0-M1-SNAPSHOT would be better?